languages/rust/oso/src/macros.rs

```diff
@@ -8,5 +8,15 @@ macro_rules! lazy_error {
 }
 
 macro_rules! check_messages {
-    ($core_obj:expr) => {};
+    ($core_obj:expr) => {
+        while let Some(message) = $core_obj.next_message() {
+            match message.kind {
+                ::polar_core::messages::MessageKind::Print => ::tracing::debug!("{}", &message.msg),
+                ::polar_core::messages::MessageKind::Warning => {
+                    ::tracing::info!("{}", &message.msg)
```

Comment:
This could probably be `warn!` to match up with it being a warning :)
```suggestion
                    ::tracing::warn!("{}", &message.msg)
```